rest/buildlogger_routes.go

```diff
@@ -151,6 +181,15 @@ func (h *logGetByTaskIDHandler) Parse(_ context.Context, r *http.Request) error
 
 // Run calls FindLogsByTaskID and returns the merged logs.
 func (h *logGetByTaskIDHandler) Run(ctx context.Context) gimlet.Responder {
+	apiLogs, err := h.sc.FindLogMetadataByTaskID(ctx, h.id)
+	if err != nil {
+		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "Error getting log metadata by task id '%s'", h.id))
+	}
+	resp := h.sc.EvergreenProxyAuthLogRead(ctx, h.userToken, h.evgURL, *apiLogs[0].Info.Project)
```

Comment:
the *apiLogs[0] looks like it could panic really easily.

Reply:
yeah the connector functions will return a not found error if there are no logs found (so this is guaranteed to always have at least one log).